Docs: fix update scripts usage (#2067)
pypa/cibuildwheel: docs/contributing.md

nox -s build                   # Make SDist and wheel
```

> More advanced users can run the update scripts. `nox -s update_pins update_constraints`.

Reviewer: ```suggestion
More advanced users can run the update scripts:
```

```console

> nox -s update_constraints

Reviewer: ```suggestion
nox -s update_constraints # update all constraints files in cibuildwheel/resources
nox -s update_pins        # update tools, python interpreters & docker images used by cibuildwheel
```

nox -s build                   # Make SDist and wheel
```

> More advanced users can run the update scripts. `update_pins` should work directly, but `update_constraints` needs all versions of Python installed. If you don't want to do that locally, a fast way to run it is to use docker to run nox:

Reviewer: ```suggestion
More advanced users can run the update scripts. `nox -s update_pins update_constraints`.
```

The `update_constraints` comment is most likely obsolete now that constraints are updated with `uv`.
The mention of `docker run ...` should be removed.
